pkg/backend/httpstate/client/client.go

```diff
@@ -211,10 +211,10 @@ func (pc *Client) GetStack(ctx context.Context, stackID StackIdentifier) (apityp
 // CreateStack creates a stack with the given cloud and stack name in the scope of the indicated project.
 func (pc *Client) CreateStack(
 	ctx context.Context, stackID StackIdentifier, cloudName string,
-	tags map[apitype.StackTagName]string) (apitype.Stack, error) {
+	tags map[apitype.StackTagName]string) (apitype.Stack, apitype.CreateStackResponseByName, error) {
```

Comment:
Putting the check outside of the client and inside the back-end felt right, so I wanted to return additional data here.  Felt reasonable to just return the entire response, but maybe that's a little bit of a factoring issue.  Just felt weird to introduce yet another type that holds two pieces of information vs just returning the entire response from the client as advisory information.

But happy to rework that if you prefer.